packages/dd-trace/test/profiling/exporters/agent.spec.js

```diff
@@ -306,86 +303,88 @@ describe('exporters/agent', function () {
     })
   })
 
-  describeOnUnix('using UDS', () => {
-    let listener
-
-    beforeEach(done => {
-      url = `${path.join(os.tmpdir(), `dd-trace-profiler-test-${Date.now()}`)}.sock`
+  if (os.platform() !== 'win32') {
```

Comment:
@uurien Might be the source of the tests failing on windows only

Reply:
I don't think so, the job that fails is running this command: `npm run test:appsec -- --coverage --nyc-arg=--include=packages/dd-trace/src/appsec/**/*.js` where `"test:appsec": "tap packages/dd-trace/test/appsec/**/*.spec.js",`

So, `packages/dd-trace/test/profiling/exporters/agent.spec.js` should be ignored.